examples/nextjs-comments-video/src/components/Mention.tsx

```diff
@@ -0,0 +1,17 @@
+import {
+  CommentBodyMentionProps,
+  Composer,
+  ComposerEditorMentionProps,
+} from "@liveblocks/react-comments/primitives";
+import { useUser } from "@/liveblocks.config";
+import styles from "./Mention.module.css";
+
+export function Mention({
+  userId,
+}: ComposerEditorMentionProps | CommentBodyMentionProps) {
+  const { user } = useUser(userId);
+
+  return (
+    <Composer.Mention className={styles.mention}>@{user.name}</Composer.Mention>
```

Comment:
We shouldn't really merge the two mentions like that at the moment, `Composer.Mention` is meant to be used specifically within `Composer`. Same for `Composer.Link`.